README.md

```diff
@@ -1 +1,22 @@
-# device-images
\ No newline at end of file
+OpenBMC power supply firmware
+
+This directory contains P11 power supply firmware images supplied by the
+vendors. Each PSU firmware should reside in a directory named as the PSU CCIN.
+The firmware image file prefix should be replaced with IBM PSU CCIN and postfix
+must remain as is. The PSU CCIN dir should contains the firmware file and
+MANIFEST file. The contents of CCIN dir can be generated in 2 ways:
+
+* - Use tool to Generate Tarball with PSU image and MANIFEST Script, the
+    tool located at phosphor-psu-code-mgmt/tools/generate-psu-tar then extract
+    the tarball into the PSU CCIN dir.
+OR 
+
+* - Add firmware file to PSU CCIN Dir and create file with the following
```

Comment:
I assume they don't want the file to contain *exactly* those contents right?  I'd think the version and extended_version would vary?  Maybe say "create a MANIFEST file with contents similar to the following:" ?

Reply:
It should say: Add firmware file to PSU CCIN Dir and create MANIFEST file with the following

Reply:
version and extended_version have same value if different phosphor-psu-code-manager will not proceed. I have asked the author about it (no reply).